maintainers/team-list.nix

```diff
@@ -197,9 +197,10 @@ with lib.maintainers;
     members = [
       reckenrode
       toonn
+      emily
```

Comment:
A minor nitpick, but you should be at the top to preserve the order. (Otherwise, I wouldn’t have added myself first if I weren’t alphabetically ordering the members list.)

Reply:
Fixed.